File: app/src/main/java/com/android/frame_master/base/BaseMvpView.java
package com.android.frame_master.base;

/**
 * @author azheng
 * @date 2018/4/24.
 * GitHub：https://github.com/RookieExaminer
 * Email：wei.azheng@foxmail.com
 * Description：
 */
public class BaseMvpView {

    /**
     * 显示加载中
     */
    public void showLoading() {
    }

    /**
     * 隐藏加载
     */
    public void hideLoading() {
    }



    /**
     * 绑定Android生命周期 防止RxJava内存泄漏
     *
     * @param <T>
     * @return
     */
    /* <T> AutoDisposeConverter<T> bindAutoDispose();*/

}

File: basic_library/src/main/java/com/frame/basic_library/base/mvp/BaseMvpView.kt
package com.frame.basic_library.base.mvp

interface BaseMvpView {
    /**
     * 显示加载中
     */
    fun showLoading()

    /**
     * 隐藏加载
     */
    fun hideLoading()
    /**
     * 绑定Android生命周期 防止RxJava内存泄漏
     *
     * @param <T>
     * @return
    </T> */
    /* <T> AutoDisposeConverter<T> bindAutoDispose();*/
}